docs/reference/settings.md

```diff
@@ -174,7 +173,7 @@ WAGTAILADMIN_RECENT_EDITS_LIMIT = 5
 
 This setting lets you change the number of items shown at 'Your most recent edits' on the dashboard.
 
-## User
+## User preferences
```

Comment:
Can the user preferences section be put under user management - seems to make sense to have these near each other. But still should be separate sections